web/www/horas/Polski/Psalterium/Chant.txt

```diff
@@ -0,0 +1,130 @@
+[sequence]
+Lauds1:1-5,Benedictus
+Lauds2:1-5
+Prima, Tertia, Sexta, Nona, Completorium
+Vespera:1-5,Magnificat
```

Comment:
This file shouldn't be necessary in any of the translations as it's the same for all languages.  It's not worth rebasing all of your changes just for this, though, so I'm happy to pull in your changes as they are, and you can delete the file later.